elife/config/etc-init-goaws-init.conf

```diff
@@ -0,0 +1,11 @@
+description "GOAWS will run a local AWS server that can be used by setting --endpoint=http://localhost:4100 to commands"
+respawn
+respawn limit 10 30
+kill timeout 70
```

Comment:
this waits for 70 seconds after `SIGTERM` before sending `SIGKILL`. This is useful for our own processes but this should terminate much faster, what about 10 seconds?